lib/tasks/assets.rake

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-EMBER_CLI = ENV["EMBER_CLI_PROD_ASSETS"] != "0"
+if !defined?(EMBER_CLI)
```

Comment:
That’s to avoid warnings about constant being already defined when code is reloaded.

Reply:
This is indeed strange, does this mean all the rake tasks are now being loaded when the Rails process is started? I don't think there is a need for us to load or autoload Rake tasks in the Rails process.

Reply:
Tasks shouldn’t be eager loaded as instructed in `application.rb` but when running `bin/rails zeitwerk:check` they are indeed loaded :thinking: